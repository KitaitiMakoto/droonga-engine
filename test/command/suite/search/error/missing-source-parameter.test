#@include fixture/user-table.jsons
{
  "type": "search",
  "dataset": "Droonga",
  "body": {
    "queries": {
      "no-source": {
      }
    }
  }
}
