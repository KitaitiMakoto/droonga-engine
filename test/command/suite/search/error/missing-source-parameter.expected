[
  "droonga.message",
  0,
  {
    "inReplyTo": null,
    "statusCode": 400,
    "type": "search.result",
    "body": {
      "name": "MissingSourceParameter",
      "message": "The query \"no-source\" has no source. Query must have a valid source.",
      "detail": {
        "queries": {
          "no-source": {
          }
        }
      }
    }
  }
]
